### PR TITLE
599 fix cob measurement noise

### PR DESCRIPTION
**Pull request by JulianHammerl**
* **Tickets addressed:** MAXGNC-599
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description

The computation of the COB error covariance was updated such that the measurement noise increases with number of pixels, instead of decreasing. Additionally, the covariance of the attitude uncertainty was included to compute the total covariance matrix.

## Verification

The UnitTest was updated.

## Documentation

The documentation was updated.

## Future work

N/A

### CODE REVIEW

File: src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.rst
Comment (thibaudteil) [inline src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.rst]: Is this 4/pi divided by the norm?

**Review by thibaudteil — APPROVED** [review]
Just a documentation comment

File: src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.rst
Comment (JulianHammerl) [reply src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.rst]: Yes, good catch. I forgot to update the documentation after we changed that computation again.

File: src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.cpp
Comment (patkenneally) [inline src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.cpp]: :large_orange_diamond: Variable name doesn't really adhere to code guidelines. Perhaps, `covarCob_C` or similar is better? Happy to discuss of course.

File: src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.cpp
Comment (patkenneally) [inline src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.cpp:224]: :large_orange_diamond: Should be squashed into the first commit.

**Review by patkenneally — APPROVED** [review]
Legend:
:small_red_triangle_down: *Issues* to address before merge
:large_orange_diamond: *Requests* that should not block merge, but should at least be discussed
:large_blue_circle: *Recommendations* that can be ignored if desired

File: src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.cpp
Comment (JulianHammerl) [reply src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.cpp]: Fixed it in the corresponding commit. Also included an additional [Refactor] commit at the end to fix the naming of older variables to adhere to code guidelines.

File: src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.cpp
Comment (JulianHammerl) [reply src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.cpp:224]: Done.

**Pull request merged**
